ccd-definition/CaseEventToComplexTypes/HomeDetailsLRspec.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "ID": "HomeDetailsLRspec",
```

Comment:
If all sections of page are mandatory and you have marked DisplayContext = Mandatory in CaseEventtoFields, my understanding is that you do not need to make any change to this file. Please can you revert the changes to this file and check if it still works.

Reply:
You're right, it works. I'll remove this file then